Model/Autobuff.cs

```diff
@@ -43,6 +43,9 @@ public _4RThread AutoBuffThread(Client c)
                     if (buffMapping.ContainsKey(status)) //CHECK IF STATUS EXISTS IN STATUS LIST AND DO ACTION
                     {
                         bmClone.Remove(status);
+                    } else if (currentStatus <= 5000)
+                    {
+                        Console.WriteLine("Unrecognized effect ID: " + currentStatus);
```

Comment:
I would suggest removing this debug line.

Reply:
Wow, sorry! ❌

Reply:
> I would suggest removing this debug line.

Fixed :)